chapters/functions.tex

```diff
@@ -1700,15 +1701,17 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 
 If {\lstinline!Inline = false!}, the model developer proposes to not inline the function.
 
+The default for inlining is tool-specific.
+
 \begin{nonnormative}
 {\lstinline!Inline = true!} is for example used in {\lstinline!Modelica.Mechanics.MultiBody.Frames!} and in functions of {\lstinline!Modelica.Media!} to have no overhead for function calls such as resolving a vector in a different coordinate system and at the same time the function can be analytically differentiated, e.g., for index reduction needed for mechanical systems.
 \end{nonnormative}
 \end{semantics}
 \end{annotationdefinition}
 
 \begin{annotationdefinition}[LateInline]
-\begin{synopsis}[grammar]\begin{lstlisting}
-"LateInline" "=" ( false | true )
+\begin{synopsis}\begin{lstlisting}
+/*literal*/ constant Boolean LateInline = false;
```

Comment:
I thought we decided to avoid writing out defaults like this, as it gives the impression that `LateInline = false` is what one is supposed to write?

Reply:
As I recall the idea was not use the form when specifying the default literal value (assuming it exists), when it is so "normal" that specifying it would look like an error (like singleInstance).

Different inlining variant and GenerateEvents don't fall under that; and is more similar to absoluteValue.

Reply:
That's not how I recall it.  For me, the main problem which was resolved when removing all those defaults was that what the reader sees first shouldn't be something one normally doesn't write when using the annotation.  This was avoided by drastically reducing the use of declaration equations for explaining defaults in case of component style annotations.

I find `LateInline` similar to `DocumentationClass`, for which the default was removed in https://github.com/modelica/ModelicaSpecification/pull/3448/commits/aa6ce474bb2c502e465d1ceb270eeba26c56ce1b.

Reply:
To me these two explanations are fairly similar.
We agree that using `DocumentationClass = false` does not really make sense at all in practice, the point is that `LateInline = false` is just a bit unusual - one can easily see someone experimenting with setting LateInline to true or false to test the result.

And most of the removed defaults in that commit weren't literal values, but explanations as defaults.

Reply:
OK, so we have different recollections of the exact formulations used in the meeting discussions behind removing literal defaults. Fine.

Now, if we think about how to write clear specification text, don't you agree that there is a problem with picking the style of presentation based on vague opinions about what is more or less unusual or odd?  … that specifying defaults for component style annotations sometimes with declaration equation and sometimes in text (even when the default corresponds to a literal value), is causing more confusion than useful brevity?